### PR TITLE
feat: point to Zulip chat rather than jabber.

### PR DESCRIPTION
**Pull request by rjsparks**

### CODE REVIEW

**Review by NGPixel — CHANGES_REQUESTED** [review]

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
This seems to be missing a migration to convert `GroupFeatures.has_default_jabber` to `GroupFeatures.has_default_chat` (or am I missing something?).

Grepping the code I find one spot where `jabber` should probably be changed to `chat` (line 732 of `ietf_filters.py`), but it's just a comment. The only other `jabber` I see is either old migrations or relates to a `jabber_room` in group extresources.

A few other questions / comments inline.

File: ietf/group/migrations/0002_groupfeatures_historicalgroupfeatures.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ class Migration(migrations.Migration):
                 ('has_nonsession_materials', models.BooleanField(default=False, verbose_name='Materials')),
                 ('has_meetings', models.BooleanField(default=False, verbose_name='Meetings')),
                 ('has_reviews', models.BooleanField(default=False, verbose_name='Reviews')),
-                ('has_default_jabber', models.BooleanField(default=False, verbose_name='Jabber')),
+                ('has_default_chat', models.BooleanField(default=False, verbose_name='Chat')),
```
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/group/migrations/0002_groupfeatures_historicalgroupfeatures.py:57]: migration is missing - the GroupFeatures model is changed.

[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/group/migrations/0002_groupfeatures_historicalgroupfeatures.py:57]: And the HistoricalGroupFeatures model was removed at group/0039, but this is a good catch, this migration shouldn't have been touched.

File: ietf/templates/group/group_about.html [hunk 2]
```diff
@@ -281,11 +282,12 @@
                     <td class="edit">
                     </td>
                     <td>
-                        <a href="https://jabber.ietf.org/jabber/logs/{{ group.acronym }}/">
-                            https://jabber.ietf.org/jabber/logs/{{ group.acronym }}/
+                        <a href="{{ group.chat_archive_url }}">
```
[the remark this answers is not in the document]
Comment (jennifer-richards) [reply ietf/templates/group/group_about.html:285]: oob, Robert indicated that it's ok to drop the logs - they're still findable and could be added via extresources if needed.

**Review by jennifer-richards — APPROVED** [review]

**Review by NGPixel — APPROVED** [review]

**Pull request merged**